api/src/main/java/org/pmiops/workbench/absorb/AbsorbService.java

```diff
@@ -3,14 +3,14 @@
 import java.util.List;
 
 public interface AbsorbService {
-  Boolean userHasLoggedIntoAbsorb(String email) throws ApiException;
+  Credentials fetchCredentials(String email) throws ApiException;
+
+  Boolean userHasLoggedIntoAbsorb(Credentials credentials);
```

Comment:
The primary change in this PR is the introduction of `fetchCredentials` and modifying `userHasLoggedIntoAbsorb` and `getActiveEnrollmentsForUser` to accept a `Credentials` object.